railties/lib/rails/generators/rails/app/templates/config/databases/postgresql.yml.tt

```diff
@@ -18,6 +18,13 @@ default: &default
   # For details on connection pooling, see Rails configuration guide
   # https://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+<% if options[:devcontainer] -%>
+  <%% if ENV["RAILS_DATABASE_HOST"] %>
+  host: <%%= ENV["RAILS_DATABASE_HOST"] %>
+  username: postgres
+  password: postgres
+  <%% end %>
+<% end %>
```

Comment:
I think, this should be used in `development` and `test` environment only.

Reply:
I think we can expect that the `DB_HOST` (I changed it from `RAILS_DATABASE_HOST` based on some feedback) env variable will not be set in production.